lib/membrane_mp4/depayloader/aac.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Membrane.MP4.Depayloader.AAC do
+  @moduledoc """
+  Depayloads AAC stream.
+  """
+  use Membrane.Filter
+
+  alias Membrane.AAC
+
+  def_input_pad :input,
+    demand_unit: :buffers,
+    demand_mode: :auto,
+    accepted_format: Membrane.MP4.Payload
+
+  def_output_pad :output,
+    demand_mode: :auto,
+    accepted_format: AAC
+
+  @impl true
+  def handle_init(_ctx, _opts) do
+    {[], %{}}
+  end
+
+  @impl true
+  def handle_stream_format(:input, stream_format, _ctx, state) do
+    content = stream_format.content
+    esds_content = get_esds_content(content.esds)
+
+    stream_format = %Membrane.AAC{
+      channels: content.channels,
+      encapsulation: :none,
+      mpeg_version: 2,
+      profile: esds_content.profile,
+      sample_rate: content.sample_rate,
+      samples_per_frame: esds_content.samples_per_frame
+    }
+
+    {[stream_format: {:output, stream_format}], state}
+  end
+
+  defp get_esds_content(esds) do
+    elementary_stream_id = 1
+    stream_priority = 0
+
+    {esds, <<>>} = unpack_esds_section(esds, 3)
+
+    <<^elementary_stream_id::16, ^stream_priority, rest::binary>> = esds
+
+    {section_4, esds_section_6} = unpack_esds_section(rest, 4)
+    {_section_6, <<>>} = unpack_esds_section(esds_section_6, 6)
```

Comment:
As far as I know that should also be ok:
```suggestion
    {<<2>>, <<>>} = unpack_esds_section(esds_section_6, 6)
```